libs/vre/shared/app-analytics/src/lib/pendo-analytics/pendo-analytics.service.ts

```diff
@@ -0,0 +1,83 @@
+import { inject, Injectable } from '@angular/core';
+import { Session, SessionService } from '@dasch-swiss/vre/shared/app-session';
+import { v5 as uuidv5 } from 'uuid';
+import {
+    DspInstrumentationConfig,
+    DspInstrumentationToken,
+} from '@dasch-swiss/vre/shared/app-config';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class PendoAnalyticsService {
+    private config: DspInstrumentationConfig = inject(DspInstrumentationToken);
+    private session: SessionService = inject(SessionService);
+    private environment: string = this.config.environment;
+
+    constructor() {
+        // FIXME: so that updates to the session state are reflected in specified user
```

Comment:
same as previous comment

Reply:
no, for future PRs. I've created issues on Linear.